frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -73,6 +73,11 @@ const getActions = (selectedObj: any) => {
   const pluginActions = actions.map((action) => (kind, ocsObj) => ({
     label: action.properties.label,
     callback: action.properties.callback(kind, ocsObj),
+    accessReview: {
+      ...action.properties.accessReview,
+      name: ocsObj.metadata.name,
+      namespace: ocsObj.metadata.namespace,
```

Comment:
I don't think we can make this assumption. The action will not always directly work on the same object, so we need to get the `accessReview` object as a callback